slice/execute.go

```diff
@@ -199,7 +199,9 @@ func (s *Split) store() error {
 		fmt.Fprintf(os.Stderr, "%d %s parsed to stdout.\n", s.fileCount, pluralize("file", s.fileCount))
 
 	default:
-		fmt.Fprintf(os.Stderr, "%d %s generated.\n", s.fileCount, pluralize("file", s.fileCount))
+		if !s.Quiet {
```

Comment:
```suggestion
		if !s.opts.Quiet {
```

Options are propagated through `s.opts`.